packages/eslint-config-react-js/index.js

```diff
@@ -22,6 +22,48 @@ module.exports = {
 		'react-hooks/exhaustive-deps': 'error',
 		'no-param-reassign': 'warn',
 		'consistent-return': 'warn',
+		'padding-line-between-statements': [
```

Comment:
If you think I missed some property, or I added it but I should not, please let me know.